include/core/parameters.h

```diff
@@ -338,6 +338,26 @@ namespace Parameters
     parse_parameters(ParameterHandler &prm);
   };
 
+  /**
+   * @brief SurfaceTensionForce - Defines the parameters for
+   * the calculation of surface tension force in the VOF solver.
+   */
+  struct SurfaceTensionForce
```

Comment:
That's a correct implementation for the time being, but surface tension will have to become a physical property also (defined in the physical properties section) so we'll have to be careful about that.

Reply:
Yes, I'll take care of it in a future PR.